warehouse/testdata/sql/7.sql

```diff
@@ -0,0 +1,23 @@
+BEGIN;
```

Comment:
`7.sql`, this naming might get unmanageable in the future. It might be better to give the name of the test file we are using it for:
```
seed_tracker_test.sql
```